scst-store/amr/auth-k8s-sa-user-defined.hbs.md

```diff
@@ -109,4 +109,4 @@ metadata:
 
 If you saved this to a file `graphql-client-rbac.yaml` you can run `kubectl apply -f graphql-client-rbac.yaml` to set everything up. If you prefer short lived service account tokens, remove the secret from the file beforehand and after creating the resources run `kubectl create token amr-graphql-view-token -n metadata-store` to create a token.
 
-Please note that if you want to configure custom service accounts even if autoconfiguratin is turned on you will need to modify adjust the naming of the resources.
\ No newline at end of file
+Please note that if you want to configure custom service accounts even if autoconfiguration is turned on you will need to adjust the naming of the resources.
```

Comment:
`Please note that if you want to configure custom service accounts even if autoconfiguration is turned on you will need to adjust the naming of the resources.` -> `Please note that if you want to configure custom service accounts even if autoconfiguration is turned on, you will need to adjust the naming of the resources.`